src/main/lrsql/admin/interceptors/lrs_management.clj

```diff
@@ -0,0 +1,30 @@
+(ns lrsql.admin.interceptors.lrs-management
+  (:require [clojure.spec.alpha :as s]
+            [io.pedestal.interceptor :refer [interceptor]]
+            [io.pedestal.interceptor.chain :as chain]
+            [lrsql.admin.protocol :as adp]))
+
+
+(def validate-delete-actor-params
+  (interceptor {:name ::validate-delete-actor-params
+                :enter (fn validate-delete-params [ctx]
+                         (let [params (get-in ctx [:request :json-params])]
+                           (if-some [err (s/explain-data (s/keys :req-un [:lrsql.spec.actor/actor-ifi]) params)]
```

Comment:
By convention we try to avoid inline spec defs like thisone. It might be good to take that `(s/keys ...)` and put it in a spec namespace, named as params for this operation (you can see other examples of this)

Reply:
Seconded. We do not want any [magic specs](https://en.wikipedia.org/wiki/Magic_number_(programming)) in our code.

Reply:
spec now lives in `lrsql.spec.admin`